### PR TITLE
GH#51152: document Single Node OKD

### PR DESCRIPTION
**Pull request by peterroth**
Version(s):
all OKD 4 versions

Issue:
GH#51152, [okd-project/planning #27](https://github.com/okd-project/planning/issues/27)

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

**Comment by peterroth** [comment]
Please check if this change is OK, @mburke5678 @vrutkovs

File: installing/installing_sno/install-sno-installing-sno.adoc [hunk 1]
```diff
@@ -6,9 +6,8 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can install {sno} using the web-based Assisted Installer and a discovery ISO that you generate using the Assisted Installer. You can also install {sno} by using `coreos-installer` to generate the installation ISO.
-
 ifndef::openshift-origin[]
+You can install {sno} using the web-based Assisted Installer and a discovery ISO that you generate using the Assisted Installer. You can also install {sno} by using `coreos-installer` to generate the installation ISO.
```
Comment (vrutkovs) [inline installing/installing_sno/install-sno-installing-sno.adoc:10]: This bit would need "how to setup Assisted Installer" section for OKD, as users can't use one from console.redhat.com.

I think a simple ref to https://github.com/openshift/assisted-service/tree/master/deploy/podman#okd-configuration would be good for a start

Comment (peterroth) [reply installing/installing_sno/install-sno-installing-sno.adoc:10]: Thanks, I wasn't aware of that document.
I just updated the file and pushed the changes.

**Comment by vrutkovs** [comment]
/ok-to-test

Looks good to me

**Comment by mburke5678** [comment]
@peterroth @vrutkovs I want to make sure what the PR is trying to do. 

* Adds a link to the upstream Install OKD using Assisted Service docs. 
* Remove the _== Installing {sno} using the Assisted Installer_ heading.
* Leaves two cross-references (but, should be combined into one _Additional references_ section).
* Removes the _Installing single-node OpenShift using the Assisted Installer_ module and two submodules.
* Removes the _Installing single-node OpenShift manually_ module and two submodules.
* Leaves the _Installing single-node OpenShift on cloud providers_ and all five submodules.
* Leaves the _Creating a bootable ISO image on a USB drive_ module. 
* Removes the final 8 modules.

Here is an internel-only (sorry!) preview: https://file.rdu.redhat.com/mburke/71324/installing/installing_sno/install-sno-installing-sno.html#install-sno-installing-sno-on-cloud-providers

Here is a screenshot that shows the new link, the two xrefs, and the TOC of remaining modules (on the right).

image


Please let me know if this is what is expected.

**Comment by peterroth** [comment]
Hello @mburke5678 ,

I can only say a partial yes, hence I'll go back into the code.

Adds a link to the upstream Install OKD using Assisted Service docs.  -- Yes, Assisted Installer isn't available for OKD
Remove the == Installing {sno} using the Assisted Installer heading.  -- Maybe a new header is required, using Assisted Service
Leaves two cross-references (but, should be combined into one Additional references section). -- Definitely no, I'll combine them
Removes the Installing single-node OpenShift using the Assisted Installer module and two submodules.  -- Yes
Removes the Installing single-node OpenShift manually module and two submodules.  -- the "Install sno manually" part isn't removed, accessible for both OCP and OKD
Leaves the Installing single-node OpenShift on cloud providers and all five submodules.  -- correct, because OKD supports those cloud providers
Leaves the Creating a bootable ISO image on a USB drive module.  -- correct
Removes the final 8 modules.  -- correct, OKD doesn't support IBM Z and IBM Power

**Comment by mburke5678** [comment]
> where can I set the reference for OKD instead of OpenShift?

That would require a new {sno} attribute in the _attributes/common-attributes.adoc file.

```
:sno: single-node OpenShift
:sno-caps: Single-node OpenShift
:sno-okd: single-node OKD
:sno-caps-okd: Single-node OKD
```

image

**Comment by peterroth** [comment]
It seems the last test failed with the below error:
```
INFO[2024-02-11T10:45:31Z] Step validate-asciidoc-openshift-docs-asciidoctor failed after 1h2m22s. 
INFO[2024-02-11T10:45:31Z] Step phase test failed after 1h2m22s.        
INFO[2024-02-11T10:45:31Z] Running multi-stage phase post               
INFO[2024-02-11T10:45:31Z] Step phase post succeeded after 0s.          
INFO[2024-02-11T10:45:31Z] Ran for 1h4m17s                              
ERRO[2024-02-11T10:45:31Z] Some steps failed:                           
ERRO[2024-02-11T10:45:31Z] 
  * could not run steps: step validate-asciidoc failed: "validate-asciidoc" test steps failed: "validate-asciidoc" pod "validate-asciidoc-openshift-docs-asciidoctor" failed: pod pending for more than 30m0s: containers have not started in 30m0.006857311s: sidecar, test: 
* Container sidecar is not ready with reason PodInitializing
* Container test is not ready with reason PodInitializing
```

The pod didn't start, it stuck in PodInitializing state for 30 min.

**Comment by peterroth** [comment]
@mburke5678 I pushed some changes to fix the issues you mentioned previously. Can you please check if the preview looks better/promising after the modifications?

**Comment by mburke5678** [comment]
@peterroth Looking good! If you can just put an asterisk * before _Install OKD using Assisted Service_ and I think we've got it!

**Comment by mburke5678** [comment]
@peterroth I noticed on issue with your edit. You need a carriage return before the asterisk. And we squash comments down to one before merging in the docs repo. But, no worry. I took the liberty of taking a copy of your PR and made the changes locally, to save any further back and forth. See :https://github.com/openshift/openshift-docs/pull/71809

**Comment by peterroth** [comment]
Sounds good. As the changes will be merged with your PR, should I close this one?

**Pull request closed without merging**